jax/_src/extend/ffi.py

```diff
@@ -197,15 +202,39 @@ def _result_avals(results: Sequence[ResultMetadata]) -> tuple[core.AbstractValue
   return tuple(avals)
 
 
+@overload
 def ffi_call(
     target_name: str,
     result_shape_dtypes: ResultMetadata | Sequence[ResultMetadata],
-    *args: ArrayLike,
+    *,
     has_side_effect: bool = False,
     vmap_method: str | None = None,
     vectorized: bool | DeprecatedArg = DeprecatedArg(),
-    **kwargs: Any,
+) -> Callable[..., Array | list[Array]]:
```

Comment:
While we are at it, can we use `Sequence[Array]` instead of `list`?

Reply:
Done. Thanks!